docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+# Developer compose file for required services
+# May be used as reference for production use but this file does not contain any best practices
+# This file has no service for the game defined as it will be started from the source code
+
+version: '3'
+services:
+
+  # authentication service
+  auth:
+    image: ghcr.io/meikellp/quantum-core-x/auth
+    restart: unless-stopped
+    ports:
+    - "11002:11002"
+    volumes:
+      - ./Executables/Auth/appsettings.json:/app/appsettings.json:ro
```

Comment:
The default appsettings in the project uses 127.0.0.1 for both database and cache which won't work here. We should add that to the documentation that you have to change it or change the default values in the repo.

Reply:
It works if you use the host mode. Maybe I used it because of that. Yeah we could use environment variables to override those values too...

Reply:
The file now contains 

```yml
    environment:
      - "Database:Host=db"
      - "Cache:Host=cache"
```

which override the values from the appsettings